README.md

```diff
@@ -7,14 +7,15 @@ Some of the pronunciations that we've heard so far.
 
 ## Kubernetes
 
-- _cube-ehr-ne-tees_
-- _koo-ber-na-tees_
-- _koo-ber-nate_
-- _koo-ber-nay-tays_
-- _koo-ber-ne-tees_
-- _koo-ber-ne-tehs_
+- _cube-ehr-ne-tees_ &nbsp;/kjʊbɜːnəti:s/
+- _koo-ber-na-tees_ &nbsp;/ku:bɜːnæti:s/
```

Comment:
I'm not sure if I transcribed this ones properly - what do you mean by -ne- and -na-? Were they really different or we could treat them as the same? Generally in English vowels tend to get reduced to a sound called "schwa" (ə) which is somewhat in the middle of the spectrum of vowels - by different people often classified as "a" or "e" (and by Poles sometimes as "y").

Reply:
‘ne’ (like in ‘never’) and ‘na’ (like in ‘naive’).

By the way, how solid are the other phonetic transcriptions? :D I wonder whether I’ll be able to add more on my own later on.